templates/js-start-scrape-single-page/src/main.js

```diff
@@ -0,0 +1,42 @@
+// Axios import - Promise based HTTP client for the browser and node.js (Read more at https://axios-http.com/docs/intro)
+import axios from "axios";
+
```

Comment:
Maybe we can squeeze the imports together? I don't think they need to have the whitespace around them.